helm/fiaas-deploy-daemon/templates/custom_rolebinding.yaml

```
@@ -0,0 +1,95 @@
+{{- if .Values.rbac.roleBinding.roles -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Values.name }}-role-grantor
+  labels:
+{{ include "fiaas-deploy-daemon.labels" . | indent 4 }}
+{{ include "fiaas-deploy-daemon.labelsOrAnnotations" .Values.rbac.roleBinding.labels | indent 4 }}
+{{- if or .Values.annotations.global .Values.rbac.roleBinding.annotations }}
+  annotations:
+{{- include "fiaas-deploy-daemon.labelsOrAnnotations" .Values.annotations.global | indent 4 }}
+{{- include "fiaas-deploy-daemon.labelsOrAnnotations" .Values.rbac.roleBinding.annotations | indent 4 }}
+{{- end }}
+rules:
+  - apiGroups:
+    - rbac.authorization.k8s.io
+    resources:
+    - roles 
+    verbs:
+    - bind
+    resourceNames:
+    {{- range $role := .Values.rbac.roleBinding.roles }}
+    - {{ $role }}
+    {{- end}}
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ .Values.name }}-rb-role-grantor
+  labels:
+{{ include "fiaas-deploy-daemon.labels" . | indent 4 }}
+{{ include "fiaas-deploy-daemon.labelsOrAnnotations" .Values.rbac.roleBinding.labels | indent 4 }}
+{{- if or .Values.annotations.global .Values.rbac.roleBinding.annotations }}
+  annotations:
+{{- include "fiaas-deploy-daemon.labelsOrAnnotations" .Values.annotations.global | indent 4 }}
+{{- include "fiaas-deploy-daemon.labelsOrAnnotations" .Values.rbac.roleBinding.annotations | indent 4 }}
+{{- end }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: {{ .Values.name }}-role-grantor
+subjects:
+- kind: ServiceAccount
+  name: {{ .Values.name }}
+  namespace: {{ .Release.Namespace }}
+{{- end }}
+
+{{- if .Values.rbac.roleBinding.clusterRoles }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Values.name }}-clusterrole-grantor
+  labels:
+{{ include "fiaas-deploy-daemon.labels" . | indent 4 }}
+{{ include "fiaas-deploy-daemon.labelsOrAnnotations" .Values.rbac.roleBinding.labels | indent 4 }}
+{{- if or .Values.annotations.global .Values.rbac.roleBinding.annotations }}
+  annotations:
+{{- include "fiaas-deploy-daemon.labelsOrAnnotations" .Values.annotations.global | indent 4 }}
+{{- include "fiaas-deploy-daemon.labelsOrAnnotations" .Values.rbac.roleBinding.annotations | indent 4 }}
+{{- end }}
+rules:
+  - apiGroups:
+    - rbac.authorization.k8s.io
+    resources:
+    - clusterroles
+    verbs:
+    - bind
+    resourceNames:
+    {{- range $clusterRole := .Values.rbac.roleBinding.cluserRoles }}
```

```suggestion
    {{- range $clusterRole := .Values.rbac.roleBinding.clusterRoles }}
```
This looks like a typo?

---

Yes, was a typo, probably it was wrong written in my values 🙈